PublicMapping/districtbuilder
# Add color picker to reference layer fly out

## Overview

This PR adds the ability to change a reference layer's color. This change is made via the reference layer flyout and is saved to the database.

### Checklist


### Demo

![color_picker_demo](https://user-images.githubusercontent.com/77936689/155018853-85435c78-e21f-4185-9353-afd198249c12.gif)

## Testing Instructions

- Start the server and open a project with existing reference layers. Verify that those existing layers display as they should and that the color picker works to change their colors.
- Create new layer, toggle it on, change its color, and verify that both the color of the icon and the layer itself change to the new color. Also verify that the selection marker in the flyout has moved to the correct color.
- Verify that on color change the layers stay in alphabetical order that no flickering occurs.

Closes #998
